kuma/settings/local.py

```diff
@@ -29,3 +29,6 @@
 # Don't cache non-versioned static files in DEBUG mode
 if DEBUG:
     WHITENOISE_MAX_AGE = 0
+
+# Send tasks to celery workers
+CELERY_ALWAYS_EAGER = False
```

Comment:
Hmm, FWIW, this may be a historical thing but in the past it was often asked to be able to run MDN without needing Celery. I guess under the 12factor app/docker pattern that would not be the case as Celery would always run, so this may totally be something that needed to happen anyway.